_posts/zh/2020-10-30-ALTF4-11.md

```diff
@@ -0,0 +1,176 @@
+---
+author: pocarski, stringweasel
+editor: Conor_, Therenas, nicgarner
+translator: Ph.X
+image:
+  url: /assets/ALTF4/11/thumbnail.jpg
+  transcription: 电影《飞屋环游记》中的房子被一堆异星工厂内燃机车吊起的画面
+title: "Alt-F4 #11 - 科学趣闻"
+lang: zh
+discuss:
+  discord: https://discord.gg/ceKebbY
+---
+
+欢迎来到第十一期的 Alt-F4，为大家带来数学分析和历史课。所以，听好了，学生们！Pocarski 亲自上阵，看看现实世界的数学是否能在 Nauvis 上发挥作用。剧透一下：不行。然后，stringweasel 再次回头看看历史，这次的重点是一些在现在已经过时的机制。
+
+## 对异星工厂世界的科学考察 <author>pocarski</author>
+
+搁置怀疑是一个众所周知的概念，但这并不妨碍我们中的一些人完全忽视它。最近，我看到了一个关于计算 Nauvis 密度的 [Reddit 帖子](https://www.reddit.com/r/factorio/comments/jaz6yl/calculating_the_density_of_nauvis/)，这让我有了一个想法，就是把异星工厂宇宙中所有荒唐而好玩的科学做个汇编。毕竟，这个社区一直都是一群拥有太多空闲时间的死宅，为什么不当一个沉迷于娱乐的死宅呢？
+
+明显的警告：这篇文章包含了过多的数学内容；如果你对数学过敏，敬请跳过这部分。
+
+### Nauvis 是一颗白矮星
+
+这是我刚才提到的 [u/DaveMcW](https://www.reddit.com/user/DaveMcW/) 的 Reddit 帖子的复述。请注意，这个理论假设 Nauvis 的昼夜周期实际上是由自转导致的，这一点我稍会提出要反对。
+
+大的天体是靠引力结合在一起的，为了使行星不被[离心力](https://zh.wikipedia.org/wiki/%E9%9B%A2%E5%BF%83%E5%8A%9B)撕裂，赤道处的重力必须大于[向心力](https://zh.wikipedia.org/wiki/%E5%90%91%E5%BF%83%E5%8A%9B)。假设一颗球形的行星，利用向心力和重力的公式，我们可以说明：
+
+$ \frac{GMm}{r^2} > mr\omega^2 $, 其中 $G$ 是[万有引力常数](https://zh.wikipedia.org/wiki/%E4%B8%87%E6%9C%89%E5%BC%95%E5%8A%9B%E5%B8%B8%E6%95%B0)，$r$ 是行星半径，$M$ 是行星质量，$\omega$ 是行星的自转角速度，$m$ 是这些力所作用的物体的质量；
+
+$ \frac{GM}{r^2} > r\omega^2 \quad\Rightarrow\quad GM > r^3\omega^2 $
+
+$ \frac{4}{3}\pi r^3 = V \quad\Rightarrow\quad r^3 = \frac{3V}{4\pi} $，其中 $V$ 是行星体积；
+
+$ GM > \frac{3V\omega^2}{4\pi} \quad\Rightarrow\quad G\frac{M}{V} > \frac{3\omega^2}{4\pi} $
+
+$ \omega = \frac{2\pi}{T} \quad\Rightarrow\quad \frac{3\omega^2}{4\pi} = \frac{3\pi}{T^2};\quad \frac{M}{V} = \rho $，其中 $ \rho $ 是行星密度；
+
+$ \frac{G\rho}{3} > \frac{\pi}{T^2} \quad\Rightarrow\quad \rho > \frac{3\pi}{GT^2} $
+
+我们最终得到了一个只需要自转周期来计算密度的方法。输入 $ \pi $ 和 $G$ 的值，以及异星工厂的昼夜循环时间（T = 25,000 ticks = 416.(6) 秒）可得：
```

Comment:
what is decimal and mil separator?
in EN: . is decimal, and , is mil.
For the ticks thingie, it's 25k ticks for ~416 secs. Please change accordingly to your locale standard

Reply:
About tick, there isn't a good translation for it, I used to use 时间片(timeslice) to explain it, but since it only appears once as an in-game term for 1/60 sec in this post, I decided to keep it untranslated, as a unit name.

Reply:
About decimal and mil separator, I believe they are correct.
According our [GB/T 15835-2011](http://openstd.samr.gov.cn/bzgk/gb/newGbInfo?hcno=F5DAC3377DA99C8D78AE66735B6359C7) ([GB/T ](https://en.wikipedia.org/wiki/Guobiao_standards) stands for national standard / recommended) 5.1.1, decimal is . and mil separator is , or just a space. Example: 19,351,235.235767

Reply:
for english, yes. is it applicable for chinese?

Reply:
erf, the GB messed me. Will check this based on this so :)
thanks!

Reply:
Yes, I'm following the Chinese national standard of general rules for writing numerals in public texts.

Reply:
TIL, French, and many [more](https://en.wikipedia.org/wiki/Decimal_separator#Countries_using_decimal_comma) countries use commas for decimal separator